<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="21766" started-at="2019-09-22T20:23:29Z" finished-at="2019-09-22T20:23:51Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="21766" started-at="2019-09-22T20:23:29Z" finished-at="2019-09-22T20:23:51Z">
      <class name="test.Amazon_com">
        <test-method status="PASS" signature="executeTest()[pri:0, instance:test.Amazon_com@7946e1f4]" name="executeTest" duration-ms="21721" started-at="2019-09-23T01:53:29Z" finished-at="2019-09-23T01:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- executeTest -->
        <test-method status="FAIL" signature="end()[pri:0, instance:test.Amazon_com@7946e1f4]" name="end" is-config="true" duration-ms="0" started-at="2019-09-23T01:53:51Z" finished-at="2019-09-23T01:53:51Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at framework.Framework.close_Browser(Framework.java:11)
	at test.Amazon_com.end(Amazon_com.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1023)
	at org.testng.TestRunner.run(TestRunner.java:628)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- test.Amazon_com -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
